### ISSUE TITLE
support all option of IKVM in the plugin

### ISSUE THREAD

**Issue by Vad1mo** [Ullink/gradle-ikvm-plugin#14]
I need to add this options `-nostdlib -reference:XYZ` and #12 asks for the  `-sharedclassloader` option. 

When looking at the plugin configuration only a handful options are supported.   Would it not make sense to support all IKVM options?

To accomplish this we need to rewrite some parts of the plugin and allow dynamic options. A map of options would make sense but I am not familiar with the Gradle plugin development.

**Comment by gluck** [comment]
`-reference` is already available no ?
I also prefer strong-naming standard arguments, to avoid confusion and simplify their use.
Why not having an extra map for non-standard/unexpected stuff though.

**Comment by Vad1mo** [comment]
so the idea is to use strong naming for everything we know/need and a key value map for anything else?

**Comment by gluck** [comment]
yes, so that the build can "understand" the goal/objective of its arguments without parsing some command-line args/map.
